app/src/main/java/ar/com/wolox/android/training/ui/login/LoginFragment.kt

```diff
@@ -34,8 +35,8 @@ class LoginFragment : WolmoFragment<LoginPresenter>(), ILoginView {
         Toast.makeText(activity?.applicationContext, "Error reading JSON, can't connect to database", Toast.LENGTH_LONG).show()
     }
 
-    override fun onLoginUserNonExistentError() {
-        vUsernameInput.error = R.string.login_error_user_non_existent.toString()
+    override fun onLoginIncorrectUserError() {
+        Toast.makeText(activity?.applicationContext, "The user you've entered is incorrect.", Toast.LENGTH_LONG).show()
```

Comment:
Be sure yo use the string xml resource here too

Reply:
This was fixed in branch "home-tabs" https://github.com/wolox-training/ph-android/pull/8